ozaria/lib/cinematic/CameraSystem.js

```diff
@@ -1,4 +1,4 @@
-import { getCamera } from '../../../schemas/selectors/cinematic'
+import { getCamera } from '../../schemas/selectors/cinematic'
```

Comment:
Is camera system and cinematic lank boss meant to be shared across all of Ozaria. If not, should it be under `ozaria/components/cinematic/common` folder?

Reply:
Same for other files below in the ozaria/lib folder

Reply:
My belief was that the components folder was specifically for `vue` files. The cinematic lank boss and other non ui cinematic systems are scoped within the `ozaria/lib/cinematic` folder. This is matching the pattern we use within app.
For example. The non-UI `world` system currently lives within `app/lib/world`. And aether is within `app/lib/aether`.

Do we want to put non UI systems in the components folder close to the vue views?
Or have a different directory structure rather than `ozaria/lib/<system>` for non UI systems?